mypy/meet.py

```diff
@@ -1,9 +1,6 @@
 from mypy.ordered_dict import OrderedDict
 from typing import List, Optional, Tuple, Callable
 
-from mypy.join import (
```

Comment:
This has to be replaced with `from mypy import join`, to prevent recursive import error.